angular.json

```diff
@@ -82,7 +82,8 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "demo-app:build"
+            "browserTarget": "demo-app:build",
+            "port": 4201
```

Comment:
Out of curiosity, why is this needed ?

Reply:
I updated this because if we're running Spike locally at the same, we'll get an error when we run the Style Guide on local because port 4200 is already in use by Spike